### PR TITLE
test commit

### PR DESCRIPTION
**Pull request by IsaiasSantanaDosSantos**

### CODE REVIEW

**Review by IsaiasSantanaDosSantos — COMMENTED** [review]
Right

**Review by IsaiasDeveloper — APPROVED** [review]
ok now

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
-    "history": "^5.3.0",
+    "history": "^4.10.1",
```
Comment (IsaiasSantanaDosSantos) [inline package.json:9]: ok

**Review by IsaiasSantanaDosSantos — COMMENTED** [review]
ok package.json

**Review by IsaiasDeveloper — APPROVED** [review]
ok

**Pull request merged**